src/script/conversation/ConversationRepository.ts

```diff
@@ -467,6 +468,35 @@ export class ConversationRepository {
     await this.userRepository.refreshUsers(unavailableUsers.map(user => user.qualifiedId));
   }
 
+  public async refreshAllConversationsUnavailableParticipants(): Promise<void> {
```

Comment:
do we need this one to be public since it's being used only in this class?

Reply:
the other one (refreshUnavailableParticipants) implemented previously by @atomrc was public so I made this public as well but we could make it private.